### PR TITLE
Easiur fix

### PR DESCRIPTION
**Pull request by adfarth**
fixes bugs in EASIUR health cost calcs

### CODE REVIEW

File: src/core/financial.jl [hunk 1]
```diff
@@ -249,6 +249,9 @@ function easiur_costs(latitude::Real, longitude::Real, grid_or_onsite::String)
     EASIUR_data = nothing
     try
         EASIUR_data = get_EASIUR2005(type, pop_year=2020, income_year=2020, dollar_year=2010)
+        if type == "p150"
```
Comment (hdunham) [inline src/core/financial.jl:252]: Debugging print statement to delete

File: src/core/financial.jl [hunk 2]
```diff
@@ -505,3 +509,5 @@ function easiur_data(; latitude::Real, longitude::Real, inflation::Real)
         end
         return response_dict
 end
+
+print("\nTEST!!!: ", easiur_data(;latitude=30.2672,longitude=-97.7431,inflation=0.025)["NOx_grid_cost_per_tonne"])
```
Comment (hdunham) [inline src/core/financial.jl:513]: another debug print

File: src/core/financial.jl [hunk 3]
```diff
@@ -400,12 +403,13 @@ function get_EASIUR2005(stack::String; pop_year::Int64=2005, income_year::Int64=
...
-    ret_map = JLD.load(fn_2005)
+    ret_map = JLD.load(fn_2005) 
+
     if pop_year != 2005
         fn_growth = joinpath(EASIUR_data_lib,"sc_growth_rate_pop2005_pop2040_$(stack).hdf5")
-        map_rate = JLD.load(fn_growth)
+        map_rate = JLD.load(fn_growth) 
         for (k,v) in map_rate
-            setindex!(ret_map, v .* (v.^(pop_year - 2005)), k)
+            setindex!(ret_map, ret_map[k] .* (v.^(pop_year - 2005)), k)
```
Comment (hdunham) [inline src/core/financial.jl:412]: why is this different?

File: src/core/financial.jl (hunk 2 above)
Comment (adfarth) [reply src/core/financial.jl]: woops sorry about that -- removed

File: src/core/financial.jl (hunk 1 above)
Comment (adfarth) [reply src/core/financial.jl]: removed, my bad!

File: src/core/financial.jl (hunk 3 above)
Comment (adfarth) [reply src/core/financial.jl:409]: `v`s are values in `map_rate` not `ret_map`.

Comment (hdunham) [reply src/core/financial.jl:409]: Oh I missed the name diff

**Review by hdunham — APPROVED** [review]

**Pull request merged**